payment_workflow/README.rst

```diff
@@ -0,0 +1,18 @@
+================
+Payment Workflow
+================
+
+Add 2 new contact type:
+* Customer Payment
+* Supplier Payment
```

Comment:
The module does not only add 2 types of contacts. The module is meant to globally improve payment management in Odoo with a workflow more similar to invoices.

The description should explain what problems this module fixes in Odoo.